### PR TITLE
Allow users to update their email address

### PR DESCRIPTION
**Pull request by taravancil**

### CODE REVIEW

File: assets/js/account-update-email.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    })
+  })
+
+  function renderErrors (json) {
+    // general error
+    $('#error-general').text(json.message)
+
+    // individual form errors
+    var details = json.details || {}
+    ;(['newPassword', 'password']).forEach(function (name) {
```
Comment (pfrazee) [inline assets/js/account-update-email.js]: I think this is wrong. Should it be newEmail, password?

**Review by pfrazee — CHANGES_REQUESTED** [review]
Looks good, just the one issue with the form

File: assets/js/account-update-email.js (hunk 1 above)
Comment (taravancil) [reply assets/js/account-update-email.js]: whoops, yes! will fix that real quick

**Comment by pfrazee** [comment]
👍

**Pull request merged**